backend/solidus_backend.gemspec

```diff
@@ -29,8 +29,8 @@ Gem::Specification.new do |s|
   s.add_dependency 'jbuilder', '~> 2.6'
   s.add_dependency 'jquery-rails'
   s.add_dependency 'kaminari', '~> 1.1'
-  s.add_dependency 'sass-rails'
-  s.add_dependency 'sass', '>= 3.5.2'
+  s.add_dependency 'sassc-rails'
+  s.add_dependency 'sassc'
```

Comment:
We should be fine with only depending on `sassc-rails`.

Reply:
Okay, thanks I will make that change and follow up by making sure the documentation is up to date with this change.